HomeW11/main.py

```diff
@@ -0,0 +1,32 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pylab
+
+plt.style.use('bmh')
+
+x = np.linspace(-11, 11, 100)
+
+pylab.subplot(2, 1, 1)
+
+a = x**3
+b = x * 50
+y = a - b
+pylab.plot(x, y);
+
+c = x**4
+c = c * -1
+d = x**2
+d = d * 88
+y = c + d
+y = y - 241
+pylab.plot(x, y);

```

Comment:
Зачем делать столько промежуточных вычислений, когда можно записать сразу в одну строчку конкретную функцию?
```suggestion
y2 = -x ** 4 + x ** 2 * 88 - 241
pylab.plot(x, y2)
```